src/Features/CSharp/Portable/QualifyMemberAccess/CSharpQualifyMemberAccessDiagnosticAnalyzer.cs

```diff
@@ -21,14 +21,20 @@ protected override bool IsAlreadyQualifiedMemberAccess(ExpressionSyntax node)
 
         // If the member is already qualified with `base.`,
         // or member is in object initialization context,
-        // or member in property or field initialization, it cannot be qualified.
+        // or member in property or field initialization,
+        // or member in constructor initializer, it cannot be qualified.
         protected override bool CanMemberAccessBeQualified(ISymbol containingSymbol, SyntaxNode node)
         {
             if (node.GetAncestorOrThis<AttributeSyntax>() != null)
             {
                 return false;
             }
 
+            if (node.GetAncestorsOrThis<ConstructorInitializerSyntax>() != null)
+            {
+                return false;
+            }
```

Comment:
Do we need to make (or at least test for) a similar issue on the VB side?

Reply:
I'll investigate and rectify if required. My VB is a little bit rusty though :-)

Reply:
no worries.  in VB you'd do: 

```vb
sub new(...)
   me.new(...)  ' or mybase.new(...)
end sub
```

Reply:
I cant seem to replicate the issue in VB and the new test seems to pass (latest commit)